src/stringIndices.js

```diff
@@ -0,0 +1,7 @@
+export const stringIndices = (string, index) => {
+  const arrString = string.split(' ')
+  if (arrString.length < index || index < 0) {
+    return ''
```

Comment:
Index in the specs would have started at 1. I do not see where that is accounted for in the code.